Foundation/NSXMLDTDNode.swift

```diff
@@ -62,7 +62,7 @@ public class NSXMLDTDNode : NSXMLNode {
         }
     } //primitive
     
-    public override init(kind: NSXMLNodeKind, options: Int) {
+    public override init(kind: NSXMLNodeKind, options: NSXMLNodeOptionRawType) {
```

Comment:
To avoid changing the API, I can cast it to that type within the function if that's what you prefer.  We've hit this problem before in https://github.com/apple/swift-corelibs-foundation/pull/169#issuecomment-166961604   Ultimately, it seems like this stuff boils down to a CF type that resolves to UInt32.  I kinda don't understand why it's promoted to a 64-bit int in Foundation.